libcaja-private/caja-undostack-manager.c

```diff
@@ -614,25 +610,11 @@ caja_undostack_manager_undo (CajaUndoStackManager * manager,
           uris = construct_gfile_list (action->destinations, action->dest_dir);
           uris = g_list_reverse (uris); // Deleting must be done in reverse
         }
-        if (priv->confirm_delete) {
-          caja_file_operations_delete (uris, NULL,
-              undo_redo_done_delete_callback, action);
-    	  g_list_free_full (uris, g_object_unref);
-        } else {
-          /* We skip the confirmation message
-           */
-          GList *f;
-          for (f = uris; f != NULL; f = f->next) {
-            char *name;
-            name = g_file_get_uri (f->data);
-            g_free (name);
-            g_file_delete (f->data, NULL, NULL);
-            g_object_unref (f->data);
-          }
-          g_list_free (uris);
-          /* Here we must do what's necessary for the callback */
-          undo_redo_done_transfer_callback (NULL, action);
-        }
+
+        caja_file_operations_delete (uris, NULL,
+            undo_redo_done_delete_callback, action);
+    	g_list_free_full (uris, g_object_unref);
```

Comment:
don't mix spaces with tabs in the line above, remove trailing space in the line below.